published/external/xdp/details/ioctlfn.h

```diff
@@ -245,6 +245,8 @@ _XdpIoctl(
                 XdpHandle, *Event, NULL, Overlapped, IoStatusBlock, Operation, InBuffer,
                 InBufferSize, OutBuffer, OutputBufferSize));
 
+#pragma warning(push)
+#pragma warning(disable: 6221)
```

Comment:
I'd rather fix the cast (even if it's adding an explicit cast) than ignore.

Reply:
I agree, though I think in this case, it might be hard to cast away the problem. If we keep the `pragma` then we must add a comment explaining what error `6221` is.